radix-engine/Cargo.toml

```diff
@@ -44,10 +44,13 @@ radix-wasm-instrument = { workspace = true,  features = ["ignore_custom_section"
 # WASM execution
 # - Wasmi is a WASM interpreter that supports WebAssembly MVP
 wasmi = {  workspace = true, features = ["no-hash-maps"] }
-
-serde_json = { workspace = true, optional = true }
 lazy_static = { workspace = true }
 
+# Serde dependencies - primarily used in the RET's receipt.
+serde = { workspace = true }
```

Comment:
I very much agree that we should move the toolkit to this repo at some point. It could be part of the mono-repo work that we want to do.

It might be a good idea to have a radix-engine-toolkit crate in here for the time being. So, I will explore that a little bit to see if we should.